CMakeLists.txt

```diff
@@ -108,11 +108,8 @@ else()
 endif()
 
 set(Libname "zenohpico")
-if(BUILD_SHARED_LIBS)
-  add_library(${Libname} SHARED)
-else()
-  add_library(${Libname} STATIC)
-endif()
+add_library(${Libname})
+add_library(zenohpico::lib ALIAS ${Libname})
```

Comment:
@DenisBiryukov91 I assume there was no reason to put the alias definition inside `add_definition` function. Let me know if I'm wrong.

Reply:
Yes, sorry, this was a typo.